.travis.yml

```diff
@@ -22,15 +25,19 @@ before_cache:
 
 matrix:
   include:
-    - env: CABALVER=head GHCVER=7.8.4
+    - env: CABALVER=head GHCVER=7.8.4 ELASTICSEARCH=1.7
```

Comment:
> Elasticsearch < 2 in general, which is the recommended version to use with Bloodhound.

In our own self interest should we test against Elasticsearch 1.5?
As this is what [AWS supports](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-version-migration.html), though [Bloodhound already tests](https://github.com/bitemyapp/bloodhound/blob/master/README.md#version-compatibility) against 1.7 (and others too).